src/bundle_list.c

```diff
@@ -151,21 +147,25 @@ static enum swupd_code list_local_bundles()
 		return SWUPD_COULDNT_LIST_DIR;
 	}
 
+	info("Installed bundles:\n");
```

Comment:
You are changing one feature on list-bundles that is unrelated to 3rd_party. Do that in a separate commit.

Reply:
👍